LICENSE

```diff
@@ -19,3 +19,29 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+******************************************************************************
+
+This project was derived from the project
+https://github.com/miguelgrinberg/oreilly-flask-apis-video
+
+The MIT License (MIT)
+
+Copyright (c) 2014 Miguel Grinberg
```

Comment:
You can't delete the copyright line. You can only add to it. You may want to add a sentence distinguishing your code from Miguel's code. Was the original code added as a distinct commit?